src/js/_enqueues/lib/comment-reply.js

```diff
@@ -203,7 +204,7 @@ window.addComment = ( function( window ) {
 		 * Third party comments systems can hook into this function via the global scope,
 		 * therefore the click event needs to reference the global scope.
 		 */
-		follow = window.addComment.moveForm(commId, parentId, respondId, postId);
+		follow = window.addComment.moveForm(commId, parentId, respondId, postId, replyTo);
```

Comment:
Let's fix the CS I messed up while changing this line.

Lazyweb question: for plugins that overwrite `moveForm` (Jetpack been one), does passing an extra attribute cause JS to explode? Hopefully that's just a PHP thing.

```suggestion
		follow = window.addComment.moveForm( commId, parentId, respondId, postId, replyTo );
```

Reply:
No, it won't explode, the extra arguments will just be ignored 😁